js/data/odata/context.js

```diff
@@ -63,6 +63,10 @@ var ODataContext = Class.inherit({
         /**
          * @name ODataContextOptions.errorHandler
          * @type function
+         * @type_function_param1 e:Error
+         * @type_function_param1_field1 httpStatus:number
+         * @type_function_param1_field2 errorDetails:boolean
```

Comment:
`errorDetails` is not `boolean`.

Reply:
It's an object
https://github.com/DevExpress/DevExtreme/blob/7a8b9942408894e28a39f70d0c49532695ddbe84/testing/tests/DevExpress.data/odataStore.tests.js#L1763-L1775